clang/test/CIR/CodeGen/tbaa-parse.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir -o %t -O1
+// RUN: cir-opt %t
```

Comment:
Because this is about fixing a parsing issue seems like a CIR to CIR test would suit this better?

Reply:
oh, and one invalid test that checks it can't parse if not TBBAA (to test the error message at the end of the function)